lib/Doctrine/Connection/Profiler.php

```diff
@@ -77,7 +77,7 @@ public function setFilterQueryType()
      * @param string $m     the name of the method
      * @param array $a      method arguments
      * @see Doctrine_EventListener
-     * @return boolean
+     * @return void
```

Comment:
This method doesn't return anything, odd that phpstan/psalm didn't catch this earlier.